### DISCUSSION TITLE
How to update the app?

### DISCUSSION THREAD

**Discussion by ifeigen** [minbrowser/min#1896] [Q&A] [answered]
Hi, I've been using the old version of min. How can I update to the latest version? Thanks :D

**Comment by PalmerAL** [comment]
Click on the update prompt in the searchbar, or go here: https://minbrowser.org/releases/, download the new version, and install it in place of the existing version.

**Comment by ifeigen** [comment, marked as answer]
Turns out, installing the app by double clicking the new setup .exe works just fine! Thank you for the answer, I'll close this discussion :D
